po/zh_CN.po

```diff
@@ -280,20 +280,14 @@ msgid ""
 msgstr "允许应用程序阻止系统响应笔记本上盖开关事件需要认证。"
 
 #: src/login/org.freedesktop.login1.policy:117
-#, fuzzy
-#| msgid "Allow applications to inhibit system handling of the power key"
 msgid "Allow applications to inhibit system handling of the reboot key"
-msgstr "允许应用程序阻止系统响应电源键"
+msgstr "允许应用程序阻止系统响应重启键"
 
 #: src/login/org.freedesktop.login1.policy:118
-#, fuzzy
-#| msgid ""
-#| "Authentication is required for an application to inhibit system handling "
-#| "of the power key."
 msgid ""
 "Authentication is required for an application to inhibit system handling of "
 "the reboot key."
-msgstr "允许应用程序阻止系统响应电源键需要认证。"
+msgstr "允许应用程序阻止系统响应重启键需要认证。"
```

Comment:
I'm not sure this translation, but this particular diff only changes "power button" to "restart key", which is appropriate in any case.